shared/common-adapters/user-proofs.native.js

```diff
@@ -1,32 +1,115 @@
+/* @flow */
+
 import React, {Component} from 'react'
 import {View, Text} from 'react-native'
 
+import openUrl from '../util/open-url'
+import * as shared from './user-proofs.shared'
+import {metaNone} from '../constants/tracker'
+import {Icon, Meta} from '../common-adapters/index'
+import {globalStyles, globalColors, globalMargins} from '../styles/style-guide'
+import {checking as proofChecking} from '../constants/tracker'
+
+import type {Props, Proof} from './user-proofs'
+
 export default class ProofsRender extends Component {
+  props: Props;
+
+  _ensureUrlProtocal (url: string): string {
```

Comment:
Desktop doesn't have this because electron will always default to using a browser to open the external links. Native tries to accommodate the links depending on their content, and so needs the hint of the http(s) to indicate that the browser should handle it.